configuration/jsr223.md

```diff
@@ -0,0 +1,210 @@
+---
+layout: documentation
+title: JSR223 Scripting
+---
+
+{% include base.html %}
+
+# JSR223 Scripting
+> Note: This feature is for advanced users who have or are willing to learn some programming skills and are comfortable working with the command line prompt of the operating system hosting openHAB.
+
+## Overview
+
+[JSR223](https://docs.oracle.com/javase/6/docs/technotes/guides/scripting/) ([spec](https://jcp.org/aboutJava/communityprocess/pr/jsr223/index.html)) is a standard scripting API for Java Virtual Machine (JVM) [languages](https://en.wikipedia.org/wiki/List_of_JVM_languages). The JVM languages provide varying levels of support for the JSR223 API and interoperability with the Java runtime. Currently the following languages are know to work well for openHAB scripting.
+
+* [Oracle Nashorn](http://www.oracle.com/technetwork/articles/java/jf14-nashorn-2126515.html) (Javascript bundled with Java)
+* [Jython](http://www.jython.org/)
+* [Apache Groovy](http://www.groovy-lang.org/)
```

Comment:
Could you add a summary phrase to Jython and Apache Groovy? Although fairly obvious, people may want to know that Jython is specifically python like.